Sources_v3/Database/DTOs/CurrentUserDTO.swift

```diff
@@ -11,7 +11,8 @@ class CurrentUserDTO: NSManagedObject {
     
     @NSManaged var unreadChannelsCount: Int16
     @NSManaged var unreadMessagesCount: Int16
-    
+    @NSManaged var lastReceivedEventDate: Date?
+
```

Comment:
nit: I wouldn't mind having some documentation here explaining why this is needed